database/ffldb/db.go

```diff
@@ -1664,6 +1700,14 @@ func (tx *transaction) writePendingAndCommit() error {
 		}
 	}
 
+	// Loop through all pending deletes and delete them.
+	for _, blockHeight := range tx.pendingBlockDeletes {
```

Comment:
Woudln't it be more efficient to find the max `blockHeight` and call `deleteBlocks` with it instead of calling it on each given `blockHeight`?